app/models/service_provider.rb

```diff
@@ -79,15 +79,10 @@ def ialmax_allowed?
   end
 
   def facial_match_ial_allowed?
-    IdentityConfig.store.biometric_ial_enabled &&
+    IdentityConfig.store.facial_match_general_availability_enabled ||
       IdentityConfig.store.allowed_biometric_ial_providers.include?(issuer)
```

Comment:
wait so we're still opting in via allowlsit? I thought the `general_availability` let us remove that

Reply:
that was the original plan, but based on the outcome of this [1-pager](https://docs.google.com/document/d/1ohF84mNojeRma5HVkxpuG94dlIHjdffz2SvdQHd6QSM/edit?tab=t.0) we need to keep the allowlist for the time being in production only. 

the general thinking seem to be that we will always need an intervention point as a control, but i think long term plan will be allowing partners to set it themselves in the partner portal so that it's a more polished process/we're not adding a giant list of issuers into our config, but i am not confident in getting that out the door in the next week.

Reply:
@zachmargolis based on the decision made in this [document](https://docs.google.com/document/d/1ohF84mNojeRma5HVkxpuG94dlIHjdffz2SvdQHd6QSM/edit?tab=t.0#heading=h.x0r0dda8iclg) we could get rid of the allowlist entirely. i am somewhat inclined to leave it for a week or two just to make sure we don't need to flip it back on, what do you think?

Reply:
thanks!